src/components/TimeseriesSearch/TimeseriesSearch.tsx

```diff
@@ -265,8 +265,8 @@ export class TimeseriesSearch extends React.Component<
         />
         <TagList
           style={{
-            ...(styles && styles.list ? styles.list : {}),
             marginTop: '8px',
```

Comment:
Why not to move `marginTop: '8px',` to `TagList` defined with styled components?

Reply:
That's a lot better. `TagList`already sets `marginTop: '16px'`, which is instantly overridden